src/arrays/ArrayUtil.java

```diff
@@ -46,50 +99,78 @@ public static boolean contains(int[] array, int value) {
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
-
-        return 0;
+    int found = -1;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value) {
+                found = i;
+                break;
+            }
+        }
+        System.out.println(found);
+        if (found == -1) {
+            return -1;
+        } else {
+            return found;
+        }
     }
 
     /**
      * Remove the element by the given index.
      * Print new array
      */
     public static void remove(int[] array, int index) {
-
+     int [] newarr = new int[array.length - 1];
+     int newarrl = array.length-(index+1);
+        System.arraycopy(array, 0, newarr, 0, index);
+         System.arraycopy(array, index + 1, newarr, index,newarrl);
+        System.out.println(Arrays.toString(array));
+           System.out.println(Arrays.toString(newarr));
     }
 
-
     /**
      * Calculate and return sum of array's elements.
      */
     public static long sum(int[] array) {
-
+    int sum = 0;
+        for (int j = 0; j < array.length; j++) {
+            sum += array[j];
+        }
+        System.out.println(sum);
         return 0;
     }
 
-
     /**
      * Get the max value from the array.
      */
     public static int getMax(int[] array) {
-
+    int Max = 0;
+        for(int i = 0; i < array.length; i++) {
+            if (array[i] > Max) {
+                Max = array[i];
+            }
+        }
+        System.out.println(Max);
         return 0;
     }
 
     /**
      * Get the minimum value from the array
      */
     public static void getMin(int[] array) {
-
+     int Min = Arrays.stream(array).min().getAsInt();
```

Comment:
it is necessary to do tasks using the skills of working with arrays, and not ready-made stream methods.

Reply:
public static void getMin(int[] array) {
     int Min = array[0];
      for(int i=1; i<array.length; i++){
          if(array[i] < Min){
              Min = array[i];
          }
      }
        System.out.println(Min);
    }